web/src/components/nav/drawer.js

```diff
@@ -0,0 +1,109 @@
+import PropTypes from 'prop-types'
+import React from 'react'
+import {
+  Divider,
+  Hidden,
+  List,
+  ListItem,
+  ListItemIcon,
+  ListItemText,
+  Drawer as MaterialDrawer,
+} from '@material-ui/core'
+import {
+  Help as HelpIcon,
+  Home as HomeIcon,
+  Input as InputIcon,
+  PersonAdd as PersonAddIcon,
+} from '@material-ui/icons/'
+import { makeStyles } from '@material-ui/core/styles'
+
+export default function Drawer({ isOpen, onDrawerClose }) {
+  const classes = useStyles()
+
+  const drawerContents = (
+    <>
+      <div className={classes.toolbar}>
+        <img
+          className={classes.logo}
+          src={'/looking-good-logo-square-black.png'}
+        />
+      </div>
+      <Divider />
+      <List>
+        <ListItem button>
+          <ListItemIcon>
+            <HomeIcon />
+          </ListItemIcon>
+          <ListItemText primary={'Home'} />
+        </ListItem>
+        <ListItem button>
+          <ListItemIcon>
+            <HelpIcon />
+          </ListItemIcon>
+          <ListItemText primary={'About'} />
+        </ListItem>
+      </List>
+      <Divider />
+      <List>
+        <ListItem button>
+          <ListItemIcon>
+            <InputIcon />
+          </ListItemIcon>
+          <ListItemText primary={'Sign In'} />
```

Comment:
Should we link this to `/login`?

Reply:
Yea I'll be setting this up, it's still a draft PR for a reason :p